MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/MuonFitPropertyBrowser.h

```diff
@@ -149,6 +190,32 @@ private slots:
   std::string m_simultaneousLabel;
   QtProperty *m_normalization;
   mutable QStringList m_normalizationValue;
```

Comment:
Is there a reason for these to be `mutable`? They're never changed inside a `const` function